docs/usage/nfd-topology-updater.md

```diff
@@ -20,5 +20,41 @@ When run as a daemonset, nodes are re-examined for the allocated resources
 (to determine the information of the allocatable resources on a per zone basis
 where a zone can be a NUMA node) at an interval specified using the
 [`-sleep-interval`](../reference/topology-updater-commandline-reference.html#-sleep-interval)
-option. The default sleep interval is set to 60s which is the the value when no
+option. The default sleep interval is set to 60s which is the value when no
 -sleep-interval is specified.
+In addition, it can avoid examining specific allocated resources
+given a configuration of resources to exclude via [`-excludeList`](../reference/topology-updater-configuration-reference.md#excludelist)
+
+## Topology-Updater Configuration
+
+NFD-Topology-Updater supports configuration through a configuration file. The
+default location is `/etc/kubernetes/node-feature-discovery/topology-updater.conf`,
+but, this can be changed by specifying the`-config` command line flag.
+unlike nfd-worker, dynamic updates are not supported ATM
```

Comment:
I'd suggest polishing the wording a bit. Also probably make this difference more obvious

```suggestion
> NOTE: unlike nfd-worker, dynamic configuration updates are not currently supported.
```